requirements/edx/base.txt

```diff
@@ -668,7 +668,7 @@ libsass==0.10.0
     #   ora2
 loremipsum==1.0.5
     # via ora2
-lti-consumer-xblock==7.2.0
+-e git+https://github.com/open-craft/xblock-lti-consumer.git@e0b64368bcf0e7d044bf9bf1a4c5ccddc61d49bc#egg=lti-consumer-xblock
```

Comment:
Just leaving an open comment to remember about changing this:
- [ ] `requirements/edx/base.txt`
- [ ] `requirements/edx/development.txt`
- [ ] `requirements/edx/testing.txt`